python/ray/tests/test_basic.py

```diff
@@ -1881,12 +1882,24 @@ def test(self):
                 [str(i) for i in gpu_ids]))
             return self.x
 
+    @ray.remote
+    class Actor2(object):
+        def __init__(self):
+            pass
+
+        def method(self):
+            pass
+
     a0 = Actor0.remote()
     ray.get(a0.test.remote())
 
     a1 = Actor1.remote()
     ray.get(a1.test.remote())
 
+    a2 = Actor2._remote()
+    ray.get(a2.method._remote())
```

Comment:
These changes look good, however, I don't think they should go in `test_gpu_ids`. Instead, can you move them to `test_submit_api`?

Reply:
"submit" API in the name `test_submit_api` here refers to invoking stuff with `_remote()`. It used to be called `_submit()`.